src/entry/controller/item.ts

```diff
@@ -66,6 +66,39 @@ class Entry extends Controller {
 
   }
 
+  async delete(ctx: Context) {
+
+    const person = await personService.findById(
+      +ctx.params.personId,
+    );
+
+    const entry = await entryService.findById(
+      person,
+      +ctx.params.entryId,
+    );
+
+    const projectUrl = ctx.request.links.get('project');
+
+    if (!projectUrl) {
+      throw new BadRequest('A link with rel "project" must be provided');
+    }
+
+    const projectId = (projectUrl.href.split('/').pop());
+    if (!projectId) {
+      throw new BadRequest('The project link must be in the format /projects/123');
+    }
```

Comment:
You don't need a Project link to do deletion. This is used to update the project associated to an entry, but doesn't make sense for deletes.